public/javascripts/searchBox.js

```diff
@@ -0,0 +1,36 @@
+var enter = function(elem, checkboxClass) {
```

Comment:
add comments to this code and other things like this so other people can understand why it's here/what it's doing.